eo-maven-plugin/src/test/java/org/eolang/maven/AssembleMojoIT.java

```diff
@@ -0,0 +1,196 @@
+                f.dependencies()
+                    .append("org.eolang", "eo-runtime", "0.51.6");
+                f.files()
+                    .file("src/main/eo/one/main.eo")
+                    .write(AssembleMojoIT.helloWorld().getBytes(StandardCharsets.UTF_8));
+                f.build()
+                    .plugins()
+                    .appendItself()
+                    .execution("tests")
+                    .goals("register", "assemble");
+                f.exec("package");
+                MatcherAssert.assertThat(
+                    String.join(
+                        " ",
+                        "AssembleMojo should have placed runtime",
+                        "library, but didn't"
+                    ),
+                    f.files().file(
+                        String.format(
+                            "target/eo/%s/org.eolang/eo-runtime/-/0.51.6/org/eolang/Phi.class",
+                            ResolveMojo.DIR
+                        )
+                    ).exists(),
+                    Matchers.is(true)
+                );
+                MatcherAssert.assertThat(
+                    String.format(
+                        "AssembleMojo should have parsed stdout object %s, but didn't",
+                        parsed
+                    ),
+                    f.files().file(parsed).exists(),
+                    Matchers.is(true)
+                );
+                MatcherAssert.assertThat(
+                    String.format(
+                        "AssembleMojo should have optimized stdout object %s, but didn't",
+                        optimized
+                    ),
+                    f.files().file(optimized).exists(),
+                    Matchers.is(true)
+                );
+                MatcherAssert.assertThat(
+                    String.format(
+                        "AssembleMojo should have pulled stdout object %s, but didn't",
+                        pulled
+                    ),
+                    f.files().file(pulled).exists(),
+                    Matchers.is(true)
+                );
+            }
+        );
+    }
+
+    @Test
+    void assemblesNotFailWithFailOnError(@Mktmp final Path temp) throws IOException {
+        final String prog = String.join(
+            "\n",
+            "+alias stdout org.eolang.io.stdout",
+            "+home https://github.com/objectionary/eo",
+            "+package test",
+            "+version 0.0.0",
+            "",
+            "[x] < wrong>",
+            "  (stdout \"Hello!\" x).print"
+        );
+        new Farea(temp).together(
+            f -> {
+                f.clean();
+                f.files()
+                    .file("src/main/eo/one/main.eo")
+                    .write(prog.getBytes(StandardCharsets.UTF_8));
+                f.build()
+                    .plugins()
+                    .appendItself()
+                    .execution("tests")
+                    .goals("register", "parse", "shake");
```

Comment:
@Suban05 let's replace `parse` and `shake` with `assemble`, I think it should no change anything